app/src/main/java/com/github/featuredetectandroid/MainActivity.kt

```diff
@@ -60,62 +48,57 @@ class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
 
-        imageViewModel.isCameraPermissionGranted = isCameraPermissionGranted()
-        tryStartCamera()
-        preferencesManager = PreferencesManager(this)
+        preferencesManager = (application as FeatureDetectApp).preferencesManager
+
+        // SuperPoint, which needs a context during initialization,
+        // does not contain it inside itself, and therefore does not require any processing
+        // in the oDestroy and onResume.
```

Comment:
1. Typo: `oDestroy`
2. Let's make it `nameToFeatureDetector`'s documentation.

Reply:
fixed